features/container/sentinel/src/main/filtered-resources/etc/org.apache.karaf.features.cfg

```diff
@@ -33,7 +33,7 @@ featuresRepositories = \
 #
```

Comment:
Considering that full smoke seems to be more flaky than before, revert this back to old state and 
keep `sentinel-core`, `opennms-health-rest-service` optional.